src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java

```diff
@@ -276,6 +276,13 @@ public void verifyMapOpened() {
     PageObjectLogging.log("verifyMapOpened", "Map was opened", true);
   }
 
+  public void verifyMapOpenedForDeleteMapTests() {
+    waitForElementVisibleByElement(mapFrame);
+    driver.switchTo().frame(mapFrame);
+    driver.switchTo().defaultContent();
+    PageObjectLogging.log("verifyMapOpenedForDeleteMapTests", "Map was opened", true);
+  }
```

Comment:
The map is inside an iframe and it is not possible to view elements within it without switching.